get.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Get Plasma Mobile
+parmalink: /get
+layout: default
+---
+ 
+Distributions offering Plasma Mobile
+====================================
+
+**Mobile**
+----------
+
+Neon based reference rootfs (recommended)
+----------------------------------------
+
+![](/img/neon.svg){:width="100"}
+
+"Official" image, based on KDE Neon. Targetted at newer devices (devices with 1GB+ RAM).
+KDE Neon itself is based upon Ubuntu 16.04 (xenial), (not Ubuntu Touch).
+This image is based on the dev-unstable branch of KDE Neon, and always ships the latest version of KDE frameworks, kwin and Plasma Mobile.
+
+**[<span class="fa fa-angle-right" /> Get it](/neon-arch-reference-rootfs)**
+
+Arch Linux based reference rootfs
+---------------------------------
+
+![](/img/archlinux.png){:width="100"}
+
+Arch Linux based image also made by the Plasma Mobile developers. Known not to be as stable as the Neon based rootfs.
+
+**[<span class="fa fa-angle-right" /> Get it](/neon-arch-reference-rootfs)**
+
+PostmarketOS
+------------
+
+![](/img/pmOS.svg){:width="100"}
+
+Distribution especially targetted at older devices, based on Alpine Linux.
+
+Their own description:
+postmarketOS (pmOS), is a touch-optimized, pre-configured Alpine Linux that can be installed on smartphones and other mobile devices. The project is at very early stages of development and is not usable for most users yet.
+
+**[<span class="fa fa-angle-right" /> Get it](https://postmarketos.org)**
+
+**Desktop**
+-----------
+
+Neon based iso image
+--------------------
+
+![](/img/neon.svg){:width="100"}
+
+Iso image using the same packages as the Neon based reference rootfs, just compiled for amd64.
```

Comment:
Article missing for "Iso image"

Also it should be ISO, instead of Iso

Better replace just with only